docs/html/reference/pip_install.rst

```diff
@@ -153,14 +153,19 @@ To interpret the requirements file in UTF-8 format add a comment
 
 The following options are supported:
 
-  *  :ref:`-i, --index-url <install_--index-url>`
-  *  :ref:`--extra-index-url <install_--extra-index-url>`
+  *  :ref:`-c, --constraint <install_--constraint>`
+  *  :ref:`-e, --editable <install_--editable>`
+  *  :ref:`-r, --requirement <install_--requirement>`
   *  :ref:`--no-index <install_--no-index>`
+  *  :ref:`-i, --index-url <install_--index-url>`
   *  :ref:`-f, --find-links <install_--find-links>`
+  *  :ref:`--extra-index-url <install_--extra-index-url>`
+  *  ``-Z, --always-unzip``
```

Comment:
This is undocumented and deprecated (see [here](https://github.com/pypa/pip/blob/9ae2e6440d9fc98e91acec0a324b410fabc5452a/src/pip/_internal/cli/cmdoptions.py#L799-L806)), so we can exclude it.

Reply:
Thanks, I noticed it is undocumented but missed the deprecation, thanks for the pointer!